tests/test_date_parser.py

```diff
@@ -304,11 +304,11 @@ def test_dates_parsing(self, date_string, expected):
         self.then_date_obj_exactly_is(expected)
 
     @parameterized.expand([
-        param('Sep 03 2014 | 4:32 pm EDT', datetime(2014, 9, 3, 21, 32)),
-        param('17th October, 2034 @ 01:08 am PDT', datetime(2034, 10, 17, 9, 8)),
-        param('15 May 2004 23:24 EDT', datetime(2004, 5, 16, 4, 24)),
+        param('Sep 03 2014 | 4:32 pm EDT', datetime(2014, 9, 3, 20, 32)),
```

Comment:
Yes @asadurski, `EDT` is 4 hours behind `UTC` because of day light savings instead of 5 hours (EST). The same applies to `PDT` case. Prior to [this change](https://github.com/scrapinghub/dateparser/pull/132/files#diff-620d77b1f1bb42b71cf2181c8b7ce193R183), these tests would pass successfully. 

Also note, EDT (-0400), instead of EST (-0500), is usually observed from last Sunday of April to Last Sunday of October.